### PR TITLE
Fit cohort includes csd and csp facilitators without assigned fit workshops

### PR DESCRIPTION
**Pull request by clareconstantine**
Follow up to [PLC-161](https://codedotorg.atlassian.net/browse/PLC-161)

Facilitators who will be doing virtual FiT Workshops don't have assigned FiT Workshops at the moment, but should still appear in the FiT Cohort view and CSV. This view should only show CSD and CSP facilitators - previously CSF facilitators were not displayed because they don't have assigned FiT Workshops, now we are explicitly only including CSD and CSP facilitators.

### CODE REVIEW

**Review by hacodeorg — APPROVED** [review]

File: dashboard/app/models/pd/application/facilitator1920_application.rb [hunk 1]
```diff
@@ -98,10 +98,10 @@ def on_successful_create
     def self.fit_cohort(applications_query = all)
       applications_query.
```
Comment (breville) [inline dashboard/app/models/pd/application/facilitator1920_application.rb:100]: Could a comment be helpful here to explain what this query is looking for?

**Review by breville — COMMENTED** [review]
Is there a test to exercise this?

File: dashboard/app/models/pd/application/facilitator1920_application.rb (hunk 1 above)
Comment (clareconstantine) [reply dashboard/app/models/pd/application/facilitator1920_application.rb:100]: updated the comment!

File: dashboard/test/controllers/api/v1/pd/applications_controller_test.rb [hunk 2]
```diff
@@ -1124,17 +1124,16 @@ class ApplicationsControllerTest < ::ActionController::TestCase
 
       # create some applications to be included in fit_cohort
       create FACILITATOR_APPLICATION_FACTORY, :locked, fit_workshop_id: fit_workshop.id, status: :accepted
-      create FACILITATOR_APPLICATION_FACTORY, :locked, fit_workshop_id: fit_workshop.id, status: :waitlisted
+      create FACILITATOR_APPLICATION_FACTORY, :locked, fit_workshop_id: fit_workshop.id, status: :withdrawn
+      # no workshop
+      create FACILITATOR_APPLICATION_FACTORY, :locked, status: :accepted
 
       #create some applications that won't be included in fit_cohort
```
Comment (breville) [inline dashboard/test/controllers/api/v1/pd/applications_controller_test.rb]: nit: missing space after #

**Pull request merged**